src/content/encrypter.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Encrypter in C
+excerpt: A beginner level encrypter in C.
+tags: C
+featuredImg: '../images/masrenko/icon.jpg'
```

Comment:
The image extension is not same as your image. Your icon has `png` format & the featured link has `jpg`.